src/Tribe/Promoter/View.php

```diff
@@ -27,6 +27,23 @@ public function add_rewrites() {
 		add_rewrite_tag( '%tribe-promoter-auth-check%', '([^&]+)' );
 	}
 
+	/**
+	 * Get the redirect URL for finishing onboarding
+	 *
+	 * @return string Redirect URL for completing onboarding.
+	 *
+	 * @since 4.9.4
+	 */
+	public function authorized_redirect_url() {
+		$url = 'https://promoter.theeventscalendar.com/welcome/review';
+
+		if ( defined( 'TRIBE_PROMOTER_AUTHORIZED_REDIRECT_URL' ) ) {
+			$url = TRIBE_PROMOTER_AUTHORIZED_REDIRECT_URL;
+		}
+
+		return $url;
```

Comment:
Maybe we can apply a filter here in case we want to override this one from a theme to test locally for example: 

```php
/**
 *
 * @param string $url
 *
 * @since TBD
 */
return apply_filters( 'tribe_promoter_authorized_redirect_url', $url );
```